### PR TITLE
[cookie_js] Handle utils and defaults

### PR DESCRIPTION
**Pull request by banyan**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
  - https://github.com/florian/cookie.js/blob/deaad9737e61c82a8b6e50053620cc426d645662/cookie.js#L9-L55
  - https://github.com/florian/cookie.js/blob/deaad9737e61c82a8b6e50053620cc426d645662/cookie.js#L70-L85
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/cookie_js/index.d.ts [hunk 1]
```diff
@@ -17,6 +17,23 @@ declare module 'cookiejs' {
 declare function cookie(key : string, fallback?: string) : string;
 declare function cookie(keys : string[], fallback?: string) : string;
 
+interface UtilStatic {
+  isArray: (value: any) => boolean;
+  isObject: (value: any) => boolean;
+  toArray: (...args: any[]) => any[];
+  getKeys: (value: {}) => any[];
+  encode: (value: string) => void;
```
Comment (slawiko) [inline types/cookie_js/index.d.ts]: As I see here https://github.com/florian/cookie.js/blob/deaad9737e61c82a8b6e50053620cc426d645662/cookie.js#L39 `encode` and `decode` functions aren't `void`. Am I wrong?

**Review by slawiko — CHANGES_REQUESTED** [review]
Actually it's nice, but I  have a small question for contributor (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23487/files#r167823468)

File: types/cookie_js/index.d.ts (hunk 1 above)
Comment (banyan) [reply types/cookie_js/index.d.ts]: @slawiko 

Thanks for the review! 🙇 You're definitely correct. 
I squshed the changes!

**Comment by banyan** [comment]
@slawiko

Please review again! 🙇

**Review by slawiko — APPROVED** [review]
It looks nice. Approved

File: types/cookie_js/index.d.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for cookie_js v1.2.2
+// Type definitions for cookie_js v1.2.3
```
Comment (DanielRosenwasser) [inline types/cookie_js/index.d.ts]: Can you enable the disabled lint rules in `tslint.json`? Typically our lint rules should pick up on an inappropriate version number.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/cookie_js/index.d.ts (hunk 2 above)
Comment (DanielRosenwasser) [reply types/cookie_js/index.d.ts]: To elaborate a bit more - we only accept a version number like `1.2`, not `v1.2.3`

Comment (banyan) [reply types/cookie_js/index.d.ts]: @DanielRosenwasser 

I should not have changed the version. sorry I didn't understand correctly, I've removed the commit. 

>Can you enable the disabled lint rules in tslint.json?

regarding this, how should I change?

Comment (banyan) [reply types/cookie_js/index.d.ts]: ping @DanielRosenwasser

**Review by DanielRosenwasser — APPROVED** [review]

**Pull request merged**